pkg/auth/handler/sso/sso.go

```diff
@@ -160,7 +160,7 @@ func (h respHandler) linkActionResp(oauthAuthInfo sso.AuthInfo, linkState sso.Li
 		return
 	}
 
-	resp = map[string]string{}
+	resp = model.NewAuthResponseWithUser(user)
```

Comment:
It seems quite useful to include the just created identity in auth response, but it would be inconsistent with other auth response, where the current session identity is returned. What do you think about it?

Reply:
I have the exact concern as yours so I didn't include it.